draft-ietf-masque-connect-udp-listen.md

```diff
@@ -213,8 +219,14 @@ COMPRESSION_CLOSE with the context ID (see {{capsulecloseformat}}) if it
 doesn't wish to support compression for the given Context ID (For example, due
 to the memory cost of establishing a list of mappings per target per client).
 If the compression was rejected, the client and proxy will instead use an
-uncompressed context ID (See {{uncompressed}}) to exhange UDP payloads for the
-given target, if those have been enabled.
+uncompressed context ID (See {{uncompressed}}) to exchange UDP payloads for the
+given target, if those have been enabled. Only one Context ID MUST be used per
+IP-port tuple. If both client and server each negotiate a Context ID for the
```

Comment:
"If both client and server each negotiate" sounds like you need to know what's going on at both endpoints. A different framing that focuses on an implementation could be: "If an endpoint detects that both itself and its peer have opened a context ID for the same tuple, the endpoint MUST close the context ID that was opened by the server."